### PR TITLE
REC-77 First version of Diversity Gini Index YAML file

### PR DESCRIPTION
**Pull request by nikosT**
The first version of the metric description has been added. See REC-77.

### CODE REVIEW

File: metric_descriptions/diversity-gini.yml [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+process:
+    - step: Clean up
+      details: >
+        Recommendations clean up; entries removal where users or services are not found in "users" or "services" files accordingly
+    - step: Services Impact
+      details: >
+        Calculation of the impact of the services, by counting how many times each service i was suggested to all possible users: count(i)
+    - step: Sort Services Impact from low to high
+      details: >
+        Sort the number of how many time each service i was suggested from the lower to the higher value, in order to apply the respective weight (i)
```
Comment (themiszamani) [inline metric_descriptions/diversity-gini.yml:31]: how many time--> how many times 
each service i --> each service (for example i)

File: metric_descriptions/diversity-gini.yml [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+    - step: Calculate the opossite of the Gini index computation
```
Comment (themiszamani) [inline metric_descriptions/diversity-gini.yml:32]: opossite --> opposite

Why do we do it? Why do we calculate the opposite?

Comment (nikosT) [reply metric_descriptions/diversity-gini.yml:32]: done

File: metric_descriptions/diversity-gini.yml (hunk 1 above)
Comment (nikosT) [reply metric_descriptions/diversity-gini.yml:31]: done

**Pull request merged**